elixir/apps/domain/lib/domain/mailer/notifications/outdated_gateway.html.eex

```diff
@@ -71,6 +71,9 @@
                   <p style="margin: 0; line-height: 24px">
                     The latest Firezone Gateway release is: <%= @latest_version %>
                   </p>
+                  <p style="margin: 0; line-height: 24px">
+                    <a href="https://www.firezone.dev/changelog?utm_source=email" target="_blank">See what's changed in this release</a>.
```

Comment:
The Gateway happens to be the first tab in the page so it is not that important but can we deep-link to it with a `#` somehow?

Should we have a more precise utm-source?

Reply:
Yeah deep link is a good idea: https://github.com/firezone/firezone/issues/8470

Hmm typically I've kept the utm_source simple so as to limit the number of possible combinations in the rollup. I'll see if there's another field for that since the referrer won't be present.